### PR TITLE
Remove unnecessary files

### PR DESCRIPTION
**Pull request by leotrs**
It seems that `perf_scenes.py` was added by someone making their own performance tests on an example scene. We can achieve the same but with a more organized test suite, in the future.

It is unclear to me exactly what `stage_scenes.py` is doing, but it is clearly only meant for the original author's private setup. If a script like this one becomes useful in the future, we can better position it inside a `scripts/` folder, for example. It would not be hard to rewrite from scratch.

### CODE REVIEW

**Comment by eulertour** [comment]
WDYT of removing the 3b1b stuff in LICENSE? Also, environment.yml was only for conda environments, which I'd suggest is too specific to be included in the upstream repo.

**Comment by eulertour** [comment]
Also, what should be done about `once_useful_constructs/`? A lot of it is sure to not even work anymore.

**Comment by leotrs** [comment]
@eulertour The NOTE.md in that folder reads

> This folder contains a collection of various things that were built for a video at some point, but were really one-off and should be given more careful consideration before being brought into the main library. In particular, there is really no guarantee of these being fully functional.

I'd say we nuke the whole folder. If anything there is of interest for a future feature, we can always check those files out from Grant's repo.

**Comment by leotrs** [comment]
RE the 3b1b stuff in LICENSE, since the folder it refers to has been deleted, we should delete it too. Yes?

File: LICENSE [hunk 1]
```diff
@@ -1,7 +1,3 @@
-All files of this project under the directory "from_3b1b" are copyright 3Blue1Brown LLC and used by permission for this project only.
-
-Any other file of this project is available under the MIT license as follow:
-
 MIT License
 
 Copyright (c) 2018 3Blue1Brown LLC
```
Comment (eulertour) [inline LICENSE:3]: This should be changed. It's probably supposed to be replaced with something else in the normal MIT license.

**Comment by leotrs** [comment]
OK I think that's about it for now.

File: LICENSE (hunk 1 above)
Comment (leotrs) [reply LICENSE:3]: Woops good catch. [Apparently it is customary](https://en.wikipedia.org/wiki/MIT_License) to add the name of the copyright holder at the top of the MIT License, even though it is very lax. What name should we use? The "manim community developers"? Or leave it blank?

Comment (eulertour) [reply LICENSE:3]: Numpy did this
https://github.com/numpy/numpy/blob/master/LICENSE.txt#L1

I guess Manim Community Developers is fine

Comment (leotrs) [reply LICENSE]: Done. I'll request others to look at it to confirm a quorum.

**Pull request merged**